azcli_conformity.sh

```diff
@@ -0,0 +1,60 @@
+# Define Variables for the Configurations
+export APP_SECRET=$(uuidgen)
+
+# Ask User Input for the App Registration Name
+echo " "
+echo "Which name do you want to give to your App Registration?"
+IFS=$'\n'
+read -r APP_NAME
+echo " "
+echo "Hello, Thank you for trying this script out.
+We're configuring your Azure Account to work with Cloud One Conformity, this might take several minutes depending of how many subscriptions you have."
+
+# Create App Registration
+az ad app create --display-name "$APP_NAME" --password $APP_SECRET --credential-description secret --required-resource-accesses @manifest.json > /dev/null
```

Comment:
♻️ Make the description a bit more descriptive?
```suggestion
az ad app create --display-name "$APP_NAME" --password $APP_SECRET --credential-description "Cloud One Conformity secret" --required-resource-accesses @manifest.json > /dev/null
```

Reply:
AZ CLI Limitation when value is anything other than "secret" it returns: 
When present, application key identifier cannot be empty and can be at most 32 bytes.
Parameter name: applicationKeyIdentifierValue

Reply:
Thanks for the explanation I didn't know about those limitations.

I think that `ccsecret` or perhaps `cocsecret` for "Cloud One Conformity" or even `tmsecret` are good options for now.

Reply:
As Tom said, limitation of Azure... Better ```cocsecret``` if we follow the new nomenclature for Cloud One Conformity, don't you think?